opensrp-simprints/src/main/java/org/smartregister/simprint/SimPrintsRegisterActivity.java

```diff
@@ -164,5 +164,12 @@ public void onClick(DialogInterface dialog, int which) {
         alertDialog.show();
     }
 
+    private boolean isRiddlerInstalled() {
+        if (SimPrintsUtils.isPackageInstalled("com.simprints.riddler", getPackageManager())) {
```

Comment:
Could also be just "return SimPrintsUtils.isPackageInstalled("com.simprints.riddler", getPackageManager()); "

Reply:
Actually that makes a lot of sense